### PR TITLE
Switch Dev Insights name in code

### PR DESCRIPTION
**Pull request by jaholme**
## Summary of the pull request
Updates all code references to PI or Project ironsides to Dev Insights.

## References and relevant issues
#3577 

## PR checklist
- [ ] Closes #xxx
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

**Review by zadesai — APPROVED** [review]

**Comment by dhoehna** [comment]
image
file renamed, but the file name looks the same.

File: tools/DevInsights/DevHome.DevInsights/Controls/PISeparator.xaml.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 using Microsoft.UI.Xaml;
 using Microsoft.UI.Xaml.Controls;
 
-namespace DevHome.PI.Controls;
+namespace DevHome.DevInsights.Controls;
```
Comment (dhoehna) [inline tools/DevInsights/DevHome.DevInsights/Controls/PISeparator.xaml.cs:7]: Should the name of the file be DevInsightsSeperator?

**Comment by dhoehna** [comment]
image

File: tools/DevInsights/DevHome.DevInsights/PI.ico
Comment (dhoehna) [inline tools/DevInsights/DevHome.DevInsights/PI.ico:1]: File name is still PI.ico

File: tools/DevInsights/DevHome.DevInsights/PIApp.xaml
Comment (dhoehna) [inline tools/DevInsights/DevHome.DevInsights/PIApp.xaml:1]: File name is still PIApp

File: tools/DevInsights/DevHome.DevInsights/Pages/ModulesPage.xaml [hunk 2]
```diff
@@ -71,7 +71,7 @@
                     </StackPanel>
                 </StackPanel>
             </ScrollViewer>
-            
+
```
Comment (dhoehna) [inline tools/DevInsights/DevHome.DevInsights/Pages/ModulesPage.xaml:74]: Remove the extra line.

File: tools/DevInsights/DevHome.DevInsights/Services/PIInsightsService.cs
Comment (dhoehna) [inline tools/DevInsights/DevHome.DevInsights/Services/PIInsightsService.cs:1]: File name is still PIInsightsDervice

File: tools/DevInsights/DevHome.DevInsights/appsettings_pi.json
Comment (dhoehna) [inline tools/DevInsights/DevHome.DevInsights/appsettings_pi.json:1]: File name is still appsettings_pi

File: tools/DevInsights/PI.ico
Comment (dhoehna) [inline tools/DevInsights/PI.ico:1]: File name is still PI.ico

File: tools/DevInsights/PI.png
Comment (dhoehna) [inline tools/DevInsights/PI.png:1]: File name is still PI.png

File: tools/DevInsights/DevHome.DevInsights/Services/DIAppInfoService.cs [hunk 3]
```diff
@@ -7,9 +7,9 @@
 using DevHome.Common.Helpers;
 using Windows.ApplicationModel;
 
-namespace DevHome.PI.Services;
+namespace DevHome.DevInsights.Services;
 
-public class PIAppInfoService
+public class DIAppInfoService
 {
     public string IconPath { get; } = Path.Combine(AppContext.BaseDirectory, "Images/PI.ico");
```
Comment (dhoehna) [inline tools/DevInsights/DevHome.DevInsights/Services/DIAppInfoService.cs:14]: Images/PI.ico

File: tools/DevInsights/DevHome.DevInsights/Views/BarWindowHorizontal.xaml [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:controls="using:Microsoft.UI.Xaml.Controls"
-    xmlns:local="using:DevHome.PI.Controls"
-    xmlns:helpers="using:DevHome.PI.Helpers"
+    xmlns:local="using:DevHome.DevInsights.Controls"
+    xmlns:helpers="using:DevHome.DevInsights.Helpers"
     mc:Ignorable="d"
     Title="" MinHeight="90" MinWidth="700" Height="90" Width="700"
     TaskBarIcon="Images/pi.ico"
```
Comment (dhoehna) [inline tools/DevInsights/DevHome.DevInsights/Views/BarWindowHorizontal.xaml:14]: Images/PI.ico

File: tools/DevInsights/DevHome.DevInsights/Views/BarWindowVertical.xaml [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
     xmlns:winex="using:WinUIEx"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-    xmlns:controls="using:DevHome.PI.Controls"
-    xmlns:helpers="using:DevHome.PI.Helpers"
+    xmlns:controls="using:DevHome.DevInsights.Controls"
+    xmlns:helpers="using:DevHome.DevInsights.Helpers"
     mc:Ignorable="d"
     Title="" MinHeight="500" MinWidth="72" MaxWidth="72" Width="72" Height="700"
     TaskBarIcon="Images/pi.ico" IsTitleBarVisible="False"
```
Comment (dhoehna) [inline tools/DevInsights/DevHome.DevInsights/Views/BarWindowVertical.xaml:13]: Images/pi.ico

File: tools/DevInsights/DevHome.DevInsights/PI.ico
Comment (jaholme) [reply tools/DevInsights/DevHome.DevInsights/PI.ico:1]: Leaving as-is until we have a new icon.

File: tools/DevInsights/DevHome.DevInsights/Controls/PISeparator.xaml.cs (hunk 1 above)
Comment (jaholme) [reply tools/DevInsights/DevHome.DevInsights/Controls/PISeparator.xaml.cs:7]: Yes

File: tools/DevInsights/DevHome.DevInsights/PIApp.xaml
Comment (jaholme) [reply tools/DevInsights/DevHome.DevInsights/PIApp.xaml:1]: Good catch

File: tools/DevInsights/DevHome.DevInsights/appsettings_pi.json
Comment (jaholme) [reply tools/DevInsights/DevHome.DevInsights/appsettings_pi.json:1]: good catch

**Review by timkur — APPROVED** [review]

**Pull request merged**